doc/examples/histology_resdnn.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+"""
+.. histology_resdnn:
+
+==================================================
+Local reconstruction using the Histological ResDNN
+==================================================
+
+A data-driven approach to modeling the non-linear mapping between observed
+DW-MRI signals and ground truth structures using sequential deep neural network
+regression with residual block deep neural network (ResDNN) [1, 2].
+
+Training was performed on two 3-D histology datasets of squirrel monkey brains
+and validated on a third. A second validation was performed on HCP datasets.
+"""
+
+import os
+
+from dipy.core.gradients import gradient_table
+from dipy.data import get_fnames, get_sphere
+from dipy.io.image import load_nifti, save_nifti
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.nn.histo_resdnn import HistoResDNN
+from dipy.reconst.shm import sh_to_sf
+from dipy.segment.mask import median_otsu
+from dipy.viz import window, actor
+import numpy as np
+import scipy.ndimage as ndi
+
+
+# Disable oneDNN warning
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+
+"""
+This ResDNN model requires single-shell data with b0, the data is fetched
+and a gradient table is constructed from bvals/bvecs.
+"""
+
+# Fetch DWI and GTAB
+hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames('stanford_hardi')
+data, affine = load_nifti(hardi_fname)
+data = np.squeeze(data)
+bvals, bvecs = read_bvals_bvecs(hardi_bval_fname, hardi_bvec_fname)
+gtab = gradient_table(bvals, bvecs)
+
+"""
+To accelerate computation a brain mask must be computed. A median-otsu filter
+isolates the main signal and then, using a floodfill, the main 'blob' is
```

Comment:
Please remove vague description about median_otsu as not necessary. Just say we use `median_otsu` to extract the brain.